templates/none/routes/index.js

```diff
@@ -0,0 +1,10 @@
+var express = require('express');
+var router = express.Router();
+
+router.get('/', function (req, res, next) {
+  res.sendFile('index.html', {
```

Comment:
It probably is, especially because I just tested this generated app and this function is ever called for `/`, since the existing `express.static` is serving it back. I think having this function when it doesn't even do anything is probably going to be super confusing...